Sources/FeaturevisorSDK/DatafileReader.swift

```diff
@@ -24,18 +30,18 @@ public class DatafileReader {
     }
 
     public func getAllAttributes() -> [Attribute] {
-        return self.attributes
+        return Array(attributes.values)
```

Comment:
It's a small improvement (if any) but you can also consider returning here sequence, so you don't need to create new Array

Reply:
Let's replace arrays with dictionaries under this PR. I believe that is not the last PR like this.